lib/ace/ace.js

```diff
@@ -36,8 +36,7 @@
 
 define(function(require, exports, module) {
 "use strict";
-
-require("./lib/fixoldbrowsers");
+"include loader_build";
```

Comment:
If I understand correctly, this is a breaking change for those who consume Ace directly via source code, right?

Reply:
I think it is not, because the things we use from fixoldbrowsers are already supported on all browsers we support, so this was only adding bloat to ace.